### PR TITLE
update

### PR DESCRIPTION
**Pull request by webelf101**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "curve-limit-order-bot-cw"
 version = "0.1.0"
-authors = ["stevenjung"]
+authors = ["Volume Fi"]
 edition = "2021"
```
Comment (taariq) [inline Cargo.toml:5]: It's not 2021. Upgrade to 2024?

**Review by taariq — APPROVED** [review]
Let's get to testing!

File: Cargo.toml (hunk 1 above)
Comment (webelf101) [reply Cargo.toml:5]: Thank you. But I think the edition we are using is 2021.

Comment (webelf101) [reply Cargo.toml:5]: It was created automatically. I didn't want to make me the alone author.

Comment (taariq) [reply Cargo.toml:5]: no worries. in future, you can just use the name volume

**Pull request merged**